tests/test_states/test_bell.py

```diff
@@ -1,58 +1,34 @@
 """Test bell."""
 import numpy as np
+import pytest
 
-from toqito.states import basis, bell
+from toqito.states import bell
 
+e_0, e_1 = np.array([[1], [0]]), np.array([[0], [1]])
 
-def test_bell_0():
-    """Generate the Bell state: `1/sqrt(2) * (|00> + |11>)`."""
-    e_0, e_1 = basis(2, 0), basis(2, 1)
-    expected_res = 1 / np.sqrt(2) * (np.kron(e_0, e_0) + np.kron(e_1, e_1))
 
-    res = bell(0)
-
-    bool_mat = np.isclose(res, expected_res)
-    np.testing.assert_equal(np.all(bool_mat), True)
-
-
-def test_bell_1():
-    """Generates the Bell state: `1/sqrt(2) * (|00> - |11>)`."""
-    e_0, e_1 = basis(2, 0), basis(2, 1)
-    expected_res = 1 / np.sqrt(2) * (np.kron(e_0, e_0) - np.kron(e_1, e_1))
-
-    res = bell(1)
-
-    bool_mat = np.isclose(res, expected_res)
-    np.testing.assert_equal(np.all(bool_mat), True)
-
-
-def test_bell_2():
-    """Generates the Bell state: `1/sqrt(2) * (|01> + |10>)`."""
-    e_0, e_1 = basis(2, 0), basis(2, 1)
-    expected_res = 1 / np.sqrt(2) * (np.kron(e_0, e_1) + np.kron(e_1, e_0))
-
-    res = bell(2)
-
-    bool_mat = np.isclose(res, expected_res)
+@pytest.mark.parametrize("bell_idx, expected_result", [
+    # 1/sqrt(2) * (|00> + |11>)
+    (0, 1 / np.sqrt(2) * (np.kron(e_0, e_0) + np.kron(e_1, e_1))),
+    # 1/sqrt(2) * (|00> - |11>)
+    (1, 1 / np.sqrt(2) * (np.kron(e_0, e_0) - np.kron(e_1, e_1))),
+    # 1/sqrt(2) * (|01> + |10>)
+    (2, 1 / np.sqrt(2) * (np.kron(e_0, e_1) + np.kron(e_1, e_0))),
+    # 1/sqrt(2) * (|01> - |10>)
+    (3, 1 / np.sqrt(2) * (np.kron(e_0, e_1) - np.kron(e_1, e_0))),
+])
+def test_bell(bell_idx, expected_result):
+    bool_mat = np.isclose(bell(bell_idx), expected_result)
     np.testing.assert_equal(np.all(bool_mat), True)
```

Comment:
```suggestion
    np.testing.assert_array_equal(bell(bell_idx), expected_result)
```
Same discussion as in `test_basis.py`.